lib/beaker/subcommands/subcommand_util.rb

```diff
@@ -86,11 +86,11 @@ def self.init_vmpooler()
         execute_rake_task("beaker_quickstart:gen_hosts[vmpooler]")
       end
 
-      # Print a message to the console and exit with 0
+      # Print a message to the console and exit with 1
       # @param [String] msg the message to print
```

Comment:
@tvpartytonight So my thinking was that this method would only used when something errors which is why I defaulted it to 1.  I can change that.  You also mentioned that the name of the method should be changed.  If we have a default of 0 for the exit code, should the name change be something like 'exit_with_msg' or do you have a better suggestion?